server/src/linter.ts

```diff
@@ -62,7 +62,7 @@ export default class Linter {
     document: LSP.TextDocument,
     folders: LSP.WorkspaceFolder[],
   ): Promise<ShellcheckResult> {
-    const args = ['--format=json1', '--external-sources', `--source-path=${this.cwd}`]
+    const args = ['--shell=bash', '--format=json1', '--external-sources', `--source-path=${this.cwd}`]
```

Comment:
Can you run `yarn lint` and recommit the file?

Reply:
Done